### PR TITLE
rm sed as external dependency for custom grafana dashboard

### PR DESCRIPTION
**Pull request by arturrez**

### CODE REVIEW

File: pkg/ssh/ssh.go [hunk 1]
```diff
@@ -264,7 +276,7 @@ func RunSSHUpdateMonitoringDashboards(host *models.Host, monitoringDashboardPath
 		if err := utils.FileCopy(utils.ExpandHome(customGrafanaDashboardPath), filepath.Join(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON)); err != nil {
 			return err
 		}
-		if err := replaceCustomVarDashboardValues(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON, chainID); err != nil {
+		if err := replaceCustomVarDashboardValues(constants.CustomGrafanaDashboardJSON, chainID); err != nil {
```
Comment (felipemadero) [inline pkg/ssh/ssh.go:279]: why monitoringDashboardPath is not needed now?

Comment (felipemadero) [reply pkg/ssh/ssh.go:279]: the file seems to be `filepath.Join(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON)` probably it is best to now assume current working dir

**Review by felipemadero — COMMENTED** [review]
couple comments then will approve

File: pkg/ssh/ssh.go (hunk 1 above)
Comment (arturrez) [reply pkg/ssh/ssh.go:279]: we don't need working dir anymore

Comment (arturrez) [reply pkg/ssh/ssh.go:279]: but you are correct.
Fixed

**Comment by arturrez** [comment]
seems to be working 
image

**Review by felipemadero — APPROVED** [review]
LGTM

**Pull request merged**